composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "symfony-bundle",
     "license": "MIT",
     "require": {
-        "php": "^8",
+        "php": "^8.1",
```

Comment:
is this ok to do if we want to support 8+? Are we going to forbid upgrades for all <8.1 (8.0) projects?